src/zksync/compile/mod.rs

```diff
@@ -287,17 +313,27 @@ impl ZkSolc {
         Ok(Self::compilers_dir()?.join(format!("{}v{}", os.get_compiler(), version)))
     }
 
+    fn solc_path(version_str: &str) -> Result<PathBuf> {
```

Comment:
I think this path is guaranteed to be unique, right?

Reply:
I think so, a path will be something like `~/.zksync/solc-linux-amd64-0.8.25-1.0.0` which don't think can clash with neither zksolc compilers in that repo non `.svm` stuff. We can totally do a better job of organizing the binaries, but it is probably something to do on top of `5.0`